### PR TITLE
Matching based on time instead of number of frames

### PR DESCRIPTION
**Pull request by alejoe91**
This fixes this issue https://github.com/SpikeInterface/spiketoolkit/issues/128

@jamesjun @magland when we merge we should propagate this change in SpikeForest

### CODE REVIEW

File: spiketoolkit/comparison/basecomparison.py [hunk 1]
```diff
@@ -1,140 +1,79 @@
...
-    
     """
-    def __init__(self, sorting1, sorting2, sorting1_name=None, sorting2_name=None, delta_frames=10, min_accuracy=0.5,
-                n_jobs=1, verbose=False):
-        self._sorting1 = sorting1
-        self._sorting2 = sorting2
+    def __init__(self, sorting1, sorting2, sorting1_name=None, sorting2_name=None, delta_time=0.3, min_accuracy=0.5,
+                 n_jobs=1, verbose=False, sampling_frequency=None):
+        self.sorting1 = sorting1
+        self.sorting2 = sorting2
```
Comment (samuelgarcia) [inline spiketoolkit/comparison/basecomparison.py]: Yeah!

File: spiketoolkit/comparison/basecomparison.py [hunk 2]
```diff
@@ -1,140 +1,79 @@
...
     def get_labels2(self, unit_id):
-        if unit_id in self._sorting1.get_unit_ids():
+        if unit_id in self.sorting1.get_unit_ids():
             return self._labels_st1[unit_id]
         else:
             raise Exception("Unit_id is not a valid unit")
 
-
-
-    def plot_confusion_matrix(self, xlabel=None, ylabel=None, ax=None, count_text=True):
```
Comment (samuelgarcia) [inline spiketoolkit/comparison/basecomparison.py]: No! Please keep it a bit more.
Please. i need a bit of time to see spike widget.
Please don't make unrelated to PR change.

**Comment by samuelgarcia** [comment]
@alejoe91 : you make too much change in the same PR.

Everything about the delta_time is OK for me.

but you should target only one thing at a time :)

I really want to have comp.plot_confusion_matrix()!!
Even if the code if elsewhere having it directly in method is cool.

What will be the dependency between spiketoolkit and spikewidget ?
I really plan to have plot facilitis in spiketoolkit (in comparison and in the study module) even if it call another module (widget).

File: spiketoolkit/comparison/basecomparison.py (hunk 2 above)
Comment (alejoe91) [reply spiketoolkit/comparison/basecomparison.py]: added back :)

**Comment by alejoe91** [comment]
@samuelgarcia I agree. I cleaned the PR and only targeter delta_frames. Also, as suggested, I rmeoved get_sorting1 and exposed sorting1 as member variable. Is that ok?

**Comment by alejoe91** [comment]
can we merge this? @samuelgarcia @magland

**Pull request merged**